dataprofiler/data_readers/data_utils.py

```diff
@@ -334,7 +330,7 @@ def reservoir(file: TextIOWrapper, sample_nrows: int) -> list:
         except StopIteration:
             break
         # Append new, replace old with dummy, and keep track of order
-        remove_index = rng.randrange(sample_nrows)
+        remove_index = rng.integers(0, sample_nrows, 1)[0]
```

Comment:
if we don't specify the size, we don't need to slice it I believe.
`rng.integers(0, sample_nrows)`

Reply:
+1

Reply:
@jacob-buehler